keyboards/arabica37/rules.mk

```diff
@@ -0,0 +1,72 @@
+# if firmware size over limit, try this option
+# CFLAGS += -flto
+
+# MCU name
+#MCU = at90usb1287
```

Comment:
I'd actually much rather have all these rules set in `keyboards/arabica37/rev1/rules.mk`, rather than here.

```suggestion
# MCU name
```